app/_hub/kong-inc/acme/index.md

```diff
@@ -0,0 +1,213 @@
+---
+name: ACME
+publisher: Kong Inc.
+version: 0.2.0
+
+source_url: https://github.com/Kong/kong-plugin-acme
+
+desc: Let's Encrypt and ACMEv2 integration with Kong
+description: |
+  This plugin allows Kong to apply cerificates from Let's Encrypt or any other ACMEv2 service and serve dynamically. Renew is handled with a configurable threshold time.
+
+type: plugin
+categories:
+  - security
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+        - 2.0.x
+    enterprise_edition:
+      compatible:
+
+params:
+  name: acme
+  api_id: false
+  service_id: false
+  route_id: false
+  consumer_id: false
+  protocols: ['http', 'https', 'tcp', 'tls', 'grpc', 'grpcs']
+  dbless_compatible: yes
+  config:
+    - name: account_email
+      required: yes
+      default:
+      description: |
+        The account identifier, can be reused in different plugin instance.
+    - name: api_uri
+      required: false
+      default: "`https://acme-v02.api.letsencrypt.org`"
+      description: |
+        The ACMEv2 API endpoint to use, user might use [Let's Encrypt staging environemnt](https://letsencrypt.org/docs/staging-environment/) during testing.
+    - name: cert_type
+      required: false
+      default: "`rsa`"
+      description: |
+        The certificate type to create, choice of `"rsa"` for RSA certificate or `"ecc"` for EC certificate.
+    - name: domains
+      required: false
+      default: "`[]`"
+      description: |
+        The list of domains to create certificate for. Wildcard domain like `*.example.com` is also supported. Regex pattern is not supported.
```

Comment:
> Wildcard domain like `*.example.com` is also supported.

I'd recommend that we should rephrase this.
This will be a source of confusion in future as some users will confuse this with wildcard certificates, which this plugin cannot support since those are based on DNS challenge.